integration-tests/utils/aws/cloudWatchGetLogs.ts

```diff
@@ -103,10 +101,6 @@ const waitForEventWithPatterns = async (
     if (logEvents.length > 1) {
       throw Error('More than 1 event matched, check filter patterns')
     }
-
-    const message = logEvents[0]?.message as string
-    console.log(`Found event: ${message}`)
```

Comment:
We'd be interested in the actual event matched for debugging purposes.

Reply:
have added those back

Reply:
I would argue the opposite. We don't need to log this event as it's massive and is one of the main causes of the noise. Might be more useful to log something generic out e.g. "Event found..." if you want to see where you are in the flow. Or somehow log out the name of the event you are searching for?